apps/cookbook/src/app/examples/virtual-scroll-example/virtual-scroll-list-example/virtual-scroll-list-example.component.html

```diff
@@ -0,0 +1,16 @@
+<kirby-page title="Items">
+  <kirby-page-content>
+    <cdk-virtual-scroll-viewport
+      minBufferPx="280"
+      maxBufferPx="560"
+      itemSize="56"
+      style="height: 280px;"
```

Comment:
should be bump the height a bit in this example, so it shows a bit more items? Something like 720px or so.

Reply:
Done